command/maintenance_window_list.go

```diff
@@ -23,5 +38,46 @@ func (c *MaintenanceWindowList) Synopsis() string {
 }
 
 func (c *MaintenanceWindowList) Run(args []string) int {
+	var includes, serviceIDs, teamIDs []string
+	var query, filter string
+	flags := c.Meta.FlagSet("maintenance-window list")
+	flags.Usage = func() { fmt.Println(c.Help())}
+	flags.Var((*ArrayFlags)(&includes), "includes", "Additional details to include (can be specified multiple times)")
+	flags.Var((*ArrayFlags)(&serviceIDs), "service-id", "Show maintenance windows for the specified services only (can be specified multiple times)")
+	flags.Var((*ArrayFlags)(&teamIDs), "team-id", "Show maintenance windows for the specified teams only (can be specified multiple times)")
+	flags.StringVar(&filter, "filter", "all", "Filter results by maintenance window state (past, future, ongoing, open, all)")
+	flags.StringVar(&query, "query", "", "Filter results showing only tags whose labels match the query")
+
+	if err := flags.Parse(args); err != nil {
+		log.Error(err)
+		return -1
+	}
+	if err := c.Meta.Setup(); err != nil {
+		log.Error(err)
+		return -1
+	}
+	client := c.Meta.Client()
+	opts := pagerduty.ListMaintenanceWindowsOptions{
+		Query:      query,
+		Includes:   includes,
+		TeamIDs:    teamIDs,
+		ServiceIDs: serviceIDs,
+		Filter:     filter,
+	}
+	mws, err := client.ListMaintenanceWindowsWithContext(context.Background(), opts)
+	if err != nil {
+		log.Error(err)
+		return -1
+	}
+	for i, mw := range mws.MaintenanceWindows {
+		fmt.Println("Entry: ", i)
+		data, err := yaml.Marshal(mw)
+		if err != nil {
+			log.Error(err)
+			return -1
+		}
+		fmt.Println(string(data))
+	}
```

Comment:
Same comment on the style nit around this block of code and the logical groups. It seems like it also applies to the rest of the files.